examples/calib_pulses.py

```diff
@@ -0,0 +1,81 @@
+r"""
+Calibrate pulses output from RF amplifier
+=========================================
+A series of betas ($\mu$s $\sqrt{W}$) are supplied to the spincore to output to
+the RF amplifier. The output of the amplifier is then captured on the GDS
+oscilloscope. It is very important to note that there MUST BE at least a 40 dBm
+attenuator between the RF amplifier output and the GDS oscilloscope input to
+avoid damaging the instrumentation! It is advised that the attenuator be
+calibrated using the GDS and AFG beforehand
+"""
+import pyspecdata as psd
+import os
+import SpinCore_pp as spc
+from datetime import datetime
+from Instruments import GDS_scope
+import numpy as np
+
+my_exp_type = "test_equipment"
+assert os.path.exists(psd.getDATADIR(exp_type=my_exp_type))
+# {{{ importing acquisition parameters
+config_dict = spc.configuration("active.ini")
+(
+    nPoints,
+    config_dict["SW_kHz"],
+    config_dict["acq_time_ms"],
+) = spc.get_integer_sampling_intervals(
+    config_dict["SW_kHz"], config_dict["acq_time_ms"]
+)
+# }}}
+# {{{ add file saving parameters to config dict
+config_dict["type"] = "pulse_calib"
+config_dict["date"] = datetime.now().strftime("%y%m%d")
+config_dict["misc_counter"] += 1
+# }}}
+# {{{ ppg
+sqrt_P = config_dict["amplitude"] * np.sqrt(75)  # we have a 75 W amplifier
+prog_beta = np.linspace(10, 100, 50, endpoint=False)
+p90s_range = prog_beta / sqrt_P
+tx_phases = np.r_[0.0, 90.0, 180.0, 270.0]
+Rx_scans = 1
+datalist = []
+with GDS_scope() as gds:
+    for index, val in enumerate(p90s_range):
+        p90 = val
+        p180 = 2 * val
```

Comment:
there should be no "p180" just call `val` `t_p` and remove the next two lines.  Do not call this p90 it is not a 90 degree pulse most or all of the time!